sirepo/feature_config.py

```diff
@@ -111,4 +99,15 @@ def _codes():
             display_test_boxes=(pkconfig.channel_in_internal_test(), bool, 'Display test boxes to visualize 3D -> 2D projections'),
         ),
     )
+    a = ALL_CODES if pkconfig.channel_in_internal_test() else NON_ALPHA_CODES
+    s = set(_cfg.sim_types) if _cfg.sim_types else set(a)
+    if _cfg.proprietary_sim_types:
```

Comment:
Why check?

Reply:
removed. old code was frozenset so was complicated to udpate.